### PR TITLE
WCAG 2.2 migration

### PR DESCRIPTION
**Pull request by WilcoFiers**
Update all relevant references to WCAG 2.2.

Needs a couple thorough reviews, but I don't believe this needs a call for review.

See also: https://github.com/act-rules/act-tools/pull/33

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/links-identical-name-equivalent-purpose-b20e66.md [hunk 1]
```diff
@@ -386,7 +386,7 @@ These `span` elements do not have a [semantic role][] of `link`. They are not va
 [matching]: #matching-characters 'Definition of matching characters'
 [presentational roles conflict resolution]: https://www.w3.org/TR/wai-aria-1.1/#conflict_resolution_presentation_none 'Presentational Roles Conflict Resolution'
```
Comment (Jym77) [inline _rules/links-identical-name-equivalent-purpose-b20e66.md]: (unrelated): 
```suggestion
[presentational roles conflict resolution]: https://www.w3.org/TR/wai-aria-1.2/#conflict_resolution_presentation_none 'Presentational Roles Conflict Resolution'
```

File: _rules/text-contrast-afw4f7.md [hunk 2]
```diff
@@ -417,20 +417,20 @@ This text is part of a [disabled][] widget because it is a child of an element w
...
 [child]: https://dom.spec.whatwg.org/#concept-tree-child 'DOM, child, 2020/07/23'
 [disabled]: #disabled-element 'Definition of Disabled'
 [flat tree]: https://drafts.csswg.org/css-scoping/#flat-tree 'CSS draft, flat tree, 2020/07/23'
 [foreground colors]: #foreground-colors-of-text 'Definition of Foreground color of text'
 [highest possible contrast]: #highest-possible-contrast 'Definition of Highest possible contrast'
-[human language]: https://www.w3.org/TR/WCAG21/#dfn-human-language-s 'WCAG 2.1, Human language'
+[human language]: https://www.w3.org/TR/WCAG22/#dfn-human-language-s 'WCAG 2.2, Human language'
 [large scale text]: #large-scale-text 'Definition of Large scale text'
 [origins]: https://www.w3.org/TR/css3-cascade/#cascading-origins 'CSS 3, origin'
 [presentational roles conflict resolution]: https://www.w3.org/TR/wai-aria-1.1/#conflict_resolution_presentation_none 'WAI-ARIA, Presentational Roles Conflict Resolution'
```
Comment (Jym77) [inline _rules/text-contrast-afw4f7.md]: (unrelated)
```suggestion
[presentational roles conflict resolution]: https://www.w3.org/TR/wai-aria-1.2/#conflict_resolution_presentation_none 'WAI-ARIA, Presentational Roles Conflict Resolution'
```

File: _rules/document-has-landmark-with-non-repeated-content-b40fd1.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ Most of the time, this rule passes by enclosing the primary content of the page
 - [Technique ARIA11: Using ARIA landmarks to identify regions of a page][tech aria11]
 - [ARIA Landmarks Example](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/index.html)
 - [CSS scoping (work in progress)](https://drafts.csswg.org/css-scoping/)
-- [The `main` role](https://www.w3.org/TR/wai-aria-1.1/#main)
+- [The `main` role](https://www.w3.org/TR/wai-aria-1.2s/#main)
```
Comment (Jym77) [inline _rules/document-has-landmark-with-non-repeated-content-b40fd1.md]:
```suggestion
- [The `main` role](https://www.w3.org/TR/wai-aria-1.2/#main)
```

File: _rules/role-attribute-valid-value-674b10.md [hunk 4]
```diff
@@ -60,7 +60,7 @@ Not every role can be used on every element. Which ARIA roles may be used on whi
 
 - [List of WAI-ARIA Roles][wai-aria role]
 - [List of Graphics ARIA Roles](https://www.w3.org/TR/graphics-aria-1.0/#role_definitions)
-- [List of DPUB ARIA Roles](https://www.w3.org/TR/dpub-aria-1.1/#role_definitions)
+- [List of DPUB ARIA Roles](https://www.w3.org/TR/dpub-aria-1.2/#role_definitions)
```
Comment (Jym77) [inline _rules/role-attribute-valid-value-674b10.md]:
```suggestion
- [List of DPUB ARIA Roles](https://www.w3.org/TR/dpub-aria-1.1/#role_definitions)
```
DPUB ARIA is still at 1.1

**Review by Jym77 — DISMISSED** [review]
Great work!
A couple of typos.

There is also the "Scrollable content can be reached with sequential focus navigation" rule which is using text from WCAG 2.1 in its examples, might as well be updated to WCAG 2.2.

I've spotted several places (more than I've noted) which link to ARIA 1.1. This should probably be taken separately anyway (#2148). (looks like this is now taken care of 😅)

**Review by carlosapaduarte — APPROVED** [review]

File: _rules/css-restrict-orientation-b33eff.md [hunk 5]
```diff
@@ -29,11 +29,11 @@ This rule applies to any [HTML element][] that is [visible](#visible) and has on
 - the CSS [rotate](https://www.w3.org/TR/css-transforms-2/#individual-transforms) property; or
 - the CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property with any of the below [transformation functions](https://www.w3.org/TR/css-transforms/#transform-functions):
 
-	- [rotate](https://www.w3.org/TR/css-transforms/#funcdef-transform-rotate)
-	- [rotate3d](https://www.w3.org/TR/css-transforms-2/#funcdef-rotate3d)
-	- [rotateZ](https://www.w3.org/TR/css-transforms-2/#funcdef-rotatez)
-	- [matrix](https://www.w3.org/TR/css-transforms/#funcdef-transform-matrix)
-	- [matrix3d](https://www.w3.org/TR/css-transforms-2/#funcdef-matrix3d)
```
Comment (WilcoFiers) [inline _rules/css-restrict-orientation-b33eff.md:36]: This gets treated as a code block, which caused tests to fail.

**Review by HelenBurge — APPROVED** [review]

**Review by sujisreerama — APPROVED** [review]

**Pull request merged**